eox_tenant/edxapp_wrapper/backends/branding_api_h_v1.py

```diff
@@ -1,5 +1,8 @@
 """ backend """
-from branding import api as branding_api  # pylint: disable=import-error
+try:
+    from branding import api as branding_api  # pylint: disable=import-error
+except ImportError:
+    branding_api = object
```

Comment:
In which case this is throwing ImportError? Shouldn't we create a new backend for the tests? or for juniper if needed?

Reply:
This is just for tests, when we added new settings a file which imports from this is loaded